config.ini

```diff
@@ -0,0 +1,17 @@
+; Schulich Delta Hermes Specification File
```

Comment:
Make this file an example file. So rename it to `config.ini.example`. This is because people are gonna change this file all time (because it's configuration) for testing and that shouldn't be tracked by git or we're gonna be commiting random non relevant changes all time.